docs/introduction/domain-settings.md

```diff
@@ -0,0 +1,157 @@
+# Domain Settings
+
+During the development with the Shopsys Framework, it is possible to meet concepts - domain, multidomain, multilanguage.
+This document explains what these terms mean and how to work with them during the development of your project.
+
+## Domain, multidomain, multilanguage
+
+- **Domain** - Domain can be understood as one instance of eshop data.
+For example, just furniture can be bough on the domain shopsys-furniture.com while only electronics can be found on the domain shopsys-electro.com.
+It is still one application with one product catalogue.
+Access to these individual domains is provided through individual url addresses.
+All domains share one common administration.
+
+- **Multidomain attribute** - A distinct value of this attribute can be set for each domain.
+An example of a multidomain attribute is a default pricing group for not logged customer.
+This pricing group can be different for each domain.
+An example of a not multidomain attribute is en ean on the product.
+This attribute is the same for each domain.
+
+- **Multilang attribute** - A distinct value of this attribute can be set for each locale.
+An example of a multilang attribute is a name of the product.
+An example of a not multilang attribute is a price of the product.
+
+- **Difference between multidomain and multilang attribute** - A value of some multilang attribute will be the same for each domain with the same locale.
+For example, when a name of the product is set as *A4tech mouse* for the locale *en* , this name of this product will be the same for each domain with the locale *en*.
+While multidomain attribute can be set to different values for different domains regardless of the locale of the domain.
+
+*Note: Demo data on the Shopsys Framework contains data only in en and cs locales*
+
+## Settings and working with domains
+
+### 1. How to create a single domain application
+
+#### 1.1 Domain configuration
+Modify the configuration of the domain in `app/config/domains.yml`.
+This configuration file contains informations about the domain ID, the domain identifier for the domain bookmarks in the administration, the domain locale.
```

Comment:
here and on one other place, there still needs to be fixed:
- ~~domain bookmarks~~ -> "domain tabs"
- "**, and** the domain locale."